src/app/components/LabeledInput/LabeledInput.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import LabeledInput from './LabeledInput';
+import UserIcon from '../Icons/UserIcon';
+import PasswordIcon from '../Icons/PasswordIcon';
+import EmailIcon from '../Icons/EmailIcon';
+import CategoryIcon from '../Icons/CategoryIcon';
+import LocationIcon from '../Icons/LocationIcon';
+import SearchIcon from '../Icons/SearchIcon';
+
+export default {
+  title: 'Component/LabeledInput',
+  component: LabeledInput,
+};
+
+export const username = (): JSX.Element => (
+  <LabeledInput
+    icon={<UserIcon />}
+    placeholder="Benutzername"
+    value=""
+    required={true}
```

Comment:
There is a short form if you like (only for boolean) 
```suggestion
    required
```

Reply:
Thanks! I changed it.